corehq/util/metrics/__init__.py

```diff
@@ -319,9 +320,7 @@ def limit_domains(domain_name):
 
     Else return __other__.  This is used to limit the number of tag combinations sent to datadog.
     """
-    if not settings.IS_SAAS_ENVIRONMENT:
-        return domain_name
-    if domain_name and domain_name in _domains_to_tag():
```

Comment:
this is the only usage of `_domains_to_tag`, so that can be removed now too (non-blocking)

Reply:
[c88f5fc](https://github.com/dimagi/commcare-hq/pull/34226/commits/c88f5fccc9c07a58f5ea60d8ace89c147784517b)